professional-services-development-guide.md

```diff
@@ -108,6 +109,13 @@ Demo flow specifications:
 * Any data created during integration tests should indicate that it was created by integration tests
 * Any data that needs to be read from the API should be flagged in the test system that it is relied upon in integration tests.
 
+## Code reliability
+* Because of cloud nature of E.io all the components must be developed with reliability in mind:
+  * Retry mechanisms must be applied. We recommend using [Exponential Backoff mechanism](https://cloud.google.com/iot/docs/how-tos/exponential-backoff)
+  * keep-alive strategy enabled (both http and https) - since Node JS Sailor 2.6.29 both http and https agents are set as global
```

Comment:
```suggestion
  * `keep-alive` strategy enabled (both http and https) - since Node JS Sailor 2.6.29 both http and https agents are set as global
```